.travis.yml

```diff
@@ -56,7 +56,13 @@ jobs:
       script:
         - if [[ "$DRIVER" == "xdebug" || "$DRIVER" == "none" ]]; then xdebug_enable; fi
         - make tests/benchmark/MutationGenerator/sources tests/benchmark/Tracing/coverage tests/benchmark/Tracing/sources
-        - if [[ "$DRIVER" != "phpdbg" ]]; then $PHPUNIT_BIN; else phpdbg -qrr $PHPUNIT_BIN; fi
+        - |
+          if [[ "$DRIVER" != "phpdbg" ]]; then
+              $PHPUNIT_BIN;
+          else
+              sed -i s/stream_get_contents/@stream_get_contents/ vendor/phpunit/phpunit/src/Util/PHP/Template/*
```

Comment:
What do you think about adding a comment above this line explaining that it related to

> Ref: sebastianbergmann/phpunit#4486

will be easier to understand what is going on here in the future

Reply:
Sure. How about now?

Reply:
:+1: